test/conformance/testdata/conformance.yaml

```diff
@@ -2033,9 +2033,12 @@
 - testname: Scheduling, HostPort matching and HostIP and Protocol not-matching
   codename: '[sig-scheduling] SchedulerPredicates [Serial] validates that there is
     no conflict between pods with same hostPort but different hostIP and protocol
-    [Conformance]'
-  description: Pods with the same HostPort value MUST be able to be scheduled to the
-    same node if the HostIP or Protocol is different.
+    [LinuxOnly] [Conformance]'
+  description: 'Pods with the same HostPort value MUST be able to be scheduled to
+    the same node if the HostIP or Protocol is different. This test is marked LinuxOnly
+    since hostNetwork is not supported on Windows TODO: Add a new e2e test to scheduler
```

Comment:
as far as I am aware HostNetwork != HostPort  

@adelina-t  was able to get this working with some changes in AzureCNI: https://github.com/Azure/azure-container-networking/pull/689.  We may need to bump our azurecni version to see these tests pass.

Reply:
Correct, the problem is with the test suite. Look at https://github.com/kubernetes/kubernetes/pull/97003/files#diff-a37213c6398de236609a3028b7594746394770ad7e5beb3630fa63b2c397b47fL701 where we create a hostNetwork pod which would fail.

Reply:
Ok, I think I get it now. The problem isn't that the hostport isn't working it is that the test uses hostnetwork for validation...